roles/docker/tasks/cleanup.yml

```diff
@@ -12,3 +12,9 @@
   become: true
   with_items:
     - "{{ docker_api_port }}"
+
+- name: cleanup contiv-compose
```

Comment:
@jojimt this will force a download even if contiv-compose version has not changed. If we are planning to maintain contiv-compose, we shall version it just like other binaries.